server.go

```diff
@@ -199,6 +199,10 @@ type server struct {
 	peerConnectedListeners    map[string][]chan<- lnpeer.Peer
 	peerDisconnectedListeners map[string][]chan<- struct{}
 
+	// TODO(yy): the Brontide.Start doesn't know this value, which means it
```

Comment:
Are these new TODOs (here and below) being fixed in follow-up PRs? Or just something you noticed during debugging that isn't a priority right now?

Reply:
I was planning to work on the other TODO once the itest fixes are finished, should be easy and quick. This one doesn't have an easy fix I think. I'm planning to move the conn-related logic into its own package so it's easier to maintain. Maybe next season?